tests/test_AllocateSizeMoreThanClusterSize.py

```diff
@@ -0,0 +1,240 @@
+"""
+Automates the following test
+
+OCS-269 - FT-OCP-Create-PV-AllocateSizeMoreThanClusterSize
+Verify a PVC creation by allocating more storage
+than what is available in Ceph
+
+https://polarion.engineering.redhat.com/polarion/#/project/
+OpenShiftContainerStorage/workitem?id=OCS-269
+
+"""
+
+import logging
+import yaml
+import os
+from utility.utils import run_cmd
+import json
+from ocs import ocp
+from openshift.dynamic import DynamicClient
+from utility import utils, templating
+from kubernetes import config
+from ocsci.enums import StatusOfTest
+import time
+
+k8s_client = config.new_client_from_config()
+dyn_client = DynamicClient(k8s_client)
+log = logging.getLogger(__name__)
+
+PVC_YAML = os.path.join("templates/ocs-deployment",
+                        "PersistentVolumeClaim.yaml")
+POD_YAML = os.path.join("templates/ocs-deployment",
+                        "Pod.yaml")
+SC_YAML = os.path.join("templates/ocs-deployment",
+                       "StorageClass.yaml")
+TEMP_PVC_YAML_FILE = '/tmp/pvc_test.yaml'
+TEMP_POD_YAML_FILE = '/tmp/pod_test.yaml'
+TEMP_SC_YAML_FILE = '/tmp/sc_test.yaml'
+
+POD = ocp.OCP(kind='Pod')
+PVC = ocp.OCP(kind='PersistentVolumeClaim')
+PV = ocp.OCP(kind='PersistentVolume')
+SC = ocp.OCP(kind='StorageClass')
+NAMESPACE = ocp.OCP(kind='Project')
+
+
+def ceph_storage_capacity(ceph_tool):
+    '''
```

Comment:
the description should be in """ instead of '''

Reply:
@keesturam you can follow this one for changing to google sphynx style docstring in pycharm:
https://stackoverflow.com/questions/18666885/custom-pycharm-docstring-stubs-i-e-for-google-docstring-or-numpydoc-formats